src/server.js

```diff
@@ -72,7 +72,7 @@ module.exports = function() {
 	manager.sockets = sockets;
 
 	var protocol = config.https.enable ? "https" : "http";
-	log.info("The Lounge v" + pkg.version + " is now running on", protocol + "://" + (config.host || "*") + ":" + config.port + "/", (config.public ? "in public mode" : "in private mode"));
+	log.info("The Lounge v" + Helper.getVersion() + " is now running on", protocol + "://" + (config.host || "*") + ":" + config.port + "/", (config.public ? "in public mode" : "in private mode"));
```

Comment:
Good job consolidating version on the command line and version in the UI, I didn't even think of that when I added it in the UI :-)